### PR TITLE
YSSY, YSBK, YSCN, YBAF, ML, SY update

### PR DESCRIPTION
**Pull request by mattkelly4**

### CODE REVIEW

**Review by zkgell — DISMISSED** [review]
Thorough description of roles without over complicating things. Looks good

**Review by tylerthetiletiler — APPROVED** [review]

File: docs/terminal/Sydney TCU/operations.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ When assigning levels on descent to aircraft, remember that you will not receive
 
 !!! example
     For an aircraft inbound from the north on the BOREE3A arrival Runway 34R, assign no lower than `A060` until south of the field, then lower as required
+
+!!! tip
+    With aircraft side by side on the RIVET and ODALE STARs, lateral separation will be lost at around 13 DME due to the converging nature of the procedures.  Consider assigning vertically-separated levels to aircraft on each STAR if they are likely to be in close lateral proximity.
```
Comment (zachbf) [inline docs/terminal/Sydney TCU/operations.md:44]: Can we add the stuff here that @LeGiraffe said, and also what J has mentioned previously?

Screen Shot 2022-11-07 at 9 00 31 am

Screen Shot 2022-11-07 at 9 00 07 am

Comment (mattkelly4) [reply docs/terminal/Sydney TCU/operations.md:44]: Yep sounds good

**Comment by mattkelly4** [comment]
Hold off reviewing for now, got a few changes to make, but I'll leave the pull request open so the comment history remains

**Review by LeGiraffe — DISMISSED** [review]

File: docs/terminal/Sydney TCU/operations.md (hunk 1 above)
Comment (alexd-332) [reply docs/terminal/Sydney TCU/operations.md:44]: if we're going down the route of including this in our TCU procedures, we'll probably need to make similar changes to the ML TCU page as well (all inbounds not via a STAR are given R090X20ML), and im sure other TCUs have similar situations

Comment (mattkelly4) [reply docs/terminal/Sydney TCU/operations.md:44]: Does ML have a similar arrangement to SY? Coord-less tracking above a certain level?

**Comment by tylerthetiletiler** [comment]
Wanna see if you can fix those merge conflicts with the terminal page Matt?

File: docs/terminal/Sydney TCU/operations.md (hunk 1 above)
Comment (zachbf) [reply docs/terminal/Sydney TCU/operations.md:44]: You could include things like "do not assign below A060" until the aircraft has passed abeam the departing runway extended centreline... (27 arbey arrival, not below 6000 until the turn the corner) and (34 arbey arrival, not below A060 until south of 27 extended centreline).

**Comment by zachbf** [comment]
> 

fixed

**Review by tylerthetiletiler — DISMISSED** [review]

**Comment by mattkelly4** [comment]
Unreal, thanks.  With those conflicts fixed, I'll spend the time to update SY and ML TCU pages with REP & discussed suggestions.  Hold off on reviewing once again while I do so.  Anything else to throw in at the same time?

File: docs/terminal/Sydney TCU/operations.md (hunk 1 above)
Comment (alexd-332) [reply docs/terminal/Sydney TCU/operations.md]: nah the R090X20ML requirement is reference the strategic departure/arrivals separation, not coord requirements between DEP and APP

**Review by alexd-332 — APPROVED** [review]

**Review by zachbf — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Pull request merged**